UpdateDotnetRuntime.ps1

```diff
@@ -0,0 +1,130 @@
+[CmdletBinding()]
```

Comment:
Maybe this file should be put in `tools` folder?

Reply:
Makes sense.. I can move it.